regression/contracts/assigns_enforce_15/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
```

Comment:
Can we add some regexes to match on the assertions that failed?

Reply:
Implemented as suggested.